js/tabulus.js

```diff
@@ -13,7 +13,19 @@ function Tabulus() {
     ** Main Function Object
     */
     function my(sel) {
-      if(!container) setup(sel);
+        if(!container) setup(sel);
+        d3.selectAll("a[data-toggle='modal']")
+            .each(function() {
+                var self = d3.select(this)
+                  , name = self.attr("data-name")
+                ;
+                self
```

Comment:
Since the legends didn't have to be generated statically per tab (and then hidden), the HTML that was creating the large tab-panes got much much smaller (to be just the panel-heading).  The buttons are in panel-body, so the set of buttons is (like the legend) now just being generated once.

thus, in order to have dynamic legends, we also need those button targets to be dynamic.